templates/repo/migrate/migrating.tmpl

```diff
@@ -32,10 +32,14 @@
 								{{end}}
 								<p id="repo_migrating_failed_error"></p>
 							</div>
-							{{if and .Failed .Permission.IsAdmin}}
+							{{if .Permission.IsAdmin}}
 								<div class="ui divider"></div>
 								<div class="item">
+									{{if .Failed}}
 									<button class="ui basic red show-modal button" data-modal="#delete-repo-modal">{{.locale.Tr "repo.settings.delete"}}</button>
+									{{else}}
+									<button class="ui basic red show-modal button" data-modal="#cancel-repo-modal">{{.locale.Tr "cancel"}}</button>
```

Comment:
So he need to cancel first and then delete?

Reply:
No if you visit the page again. I do not want to make the page too complex.